sources/api/datastore/src/serialization/pairs.rs

```diff
@@ -516,20 +515,22 @@ mod test {
             list: vec![3, 4, 5],
             boolean: true,
         };
-        let keys = to_pairs(&b).unwrap();
+        let j = serde_json::to_value(b).unwrap();
+        let keys = to_pairs(&j).unwrap();
         assert_eq!(
             keys,
             hashmap!(
-                key!("B.list") => "[3,4,5]".to_string(),
-                key!("B.boolean") => "true".to_string(),
+                key!("list") => "[3,4,5]".to_string(),
+                key!("boolean") => "true".to_string(),
```

Comment:
oh yeah, meant to leave a note on this in the PR. we had removed the cases where the serializer depended on the struct name as part of the settings plugins work because I couldn't find a way to rename `serde_json::Value`, so this shouldn't actually cause a change in behavior in practice